package.json

```diff
@@ -25,10 +25,10 @@
     "watch": "npm run mocha -- --watch tests/src",
     "pretest": "linklocal",
     "posttest": "eslint .",
-    "mocha": "cross-env BABEL_ENV=test NODE_PATH=./src nyc -s mocha -R dot --recursive -t 5s",
+    "mocha": "cross-env BABEL_ENV=src nyc -s mocha -R dot --recursive -t 5s",
     "tests-only": "npm run mocha tests/src",
     "test": "npm run tests-only",
-    "test-compiled": "npm run prepublish && NODE_PATH=./lib mocha --compilers js:babel-register --recursive tests/src",
+    "test-compiled": "npm run prepublish && BABEL_ENV=lib mocha --compilers js:babel-register --recursive tests/src",
```

Comment:
i *very much* like avoiding use of the terrible and horrific `NODE_PATH`, and using babel for this instead